packages/components/src/Dialog/useDialog.tsx

```diff
@@ -220,9 +188,8 @@ export const useDialog = ({
         <RenderSurface
           aria-busy={busy ? true : undefined}
           className={className}
-          style={surfaceStyles}
-          width={width}
-          maxWidth={maxWidth}
+          placement={placement as any}
```

Comment:
You could do this to at least get rid of the `any` although genericizing some of the interfaces might be more accurate (also might be overkill).

```suggestion
          placement={placement as DialogPlacements}
```